docs/DEPLOYMENT.md

```diff
@@ -146,22 +146,68 @@ Follow the steps below to set up the staging environment on AWS.
+      1. Go to the ElastiCache dashboard on AWS. On the top right corner, select the region where you want to launch the Redis Cluster and click on **Create** button
+      2. Select **Redis** as your Cluster engine. In the checkbox below, choosing between *Cluster mode enabled* or *Cluster mode disabled* is your choice. Plio uses *Cluster mode disabled*. More details on cluster mode [here](https://docs.aws.amazon.com/AmazonElastiCache/latest/red-ug/Replication.Redis-RedisCluster.html)
+      3. Choose **Amazon Cloud** as your location in the **Location** section
+      4. Fill all the details in the **Redis Details** section. Some pointers below:
+         1. Choose a name for your Redis Cluster, e.g. `plio-redis-staging`. Description is optional
+         2. For **Engine version compatibility**, it's recommended to choose the latest one.
+         3. For **Port**, enter the port you want for the instance. `6379` is the default port for redis and is recommended.
+         4. For **Parameter Group**, nothing needs to be changed. This gets decided when choosing between cluster mode enabled or disabled
+         5. Choose your **Node Type**. Plio uses `cache.t2.micro`. The size of the node should depend on the workload of your apps in production. For more details, see [this](https://docs.aws.amazon.com/AmazonElastiCache/latest/red-ug/nodes-select-size.html#CacheNodes.SelectSize)
+         6. For **Number of replicas**, choose the number of read replicas that you want for this cluster. It's recommended to have 1 or more replicas. If you choose to have 1 or more replicas, make sure you select *Multi AZ* checkbox. It provides high availability and failover support. More details [here](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Concepts.MultiAZ.html)
+         7. Select **Advanced Redis settings** and fill in the details as mentioned below:
+            1. For *Subnet group* you can create a new group or use an existing one. For staging, subnets were created in step `2.vii`. You can create a new subnet group, name it accordingly and add the subnets (created in step `2.vii`) to that group. A VPC was also created in step `2`. Add that VPC ID in the appropriate selectbox. *NOTE*: If you selected Multi-AZ checkbox in step `11.iv.f`, then the subnet group will require two or more subnets in the group. For example, in staging environment for Plio, three subnets have been created and added to the group.
+            2. For *Availability zone(s)*, choose *No preference* – ElastiCache chooses the Availability Zones for your cluster's nodes. If required, configure your own availability zones. For more info, see [this](https://docs.aws.amazon.com/AmazonElastiCache/latest/red-ug/RegionsAndAZs.html)
+            3. For Security groups, choose the security groups that you want for this cluster. A security group acts as a firewall to control network access to your cluster. You can choose use the Default security group for your VPC or create a new one.
```

Comment:
We should not go into explaining what each feature of AWS does as Vaibhav has mentioned - that's the burden of the developer.
```suggestion
            3. For Security groups, choose the Default security group for your VPC.
```